pkg/services/alerting/notifiers/email.go

```diff
@@ -52,7 +52,7 @@ func NewEmailNotifier(model *m.AlertNotification) (alerting.Notifier, error) {
 	})
 
 	return &EmailNotifier{
-		NotifierBase: NewNotifierBase(model.Id, model.IsDefault, model.Name, model.Type, model.Settings),
+		NotifierBase: NewNotifierBase(model.Id, model.IsDefault, model.Name, model.Type, model.NotifyOnce, model.Frequency, model.Settings),
```

Comment:
I think we should look into moving the creation of NotifierBase out of the specific notifiers so that we don't have to make modifications to these specific notifiers each time we make changes to the base.

Reply:
agree!